modules/ROOT/pages/digger/index.adoc

```diff
@@ -3,10 +3,14 @@ include::{partialsdir}/attributes.adoc[]
 //':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
 :context: con_introduction-to-build
 
-[id='{context}_con_introduction-to-build']
-= About {digger-service}
+[[overview]]
+== Overview
```

Comment:
not the pattern we're using 
https://docs.google.com/presentation/d/1wxHPTo6mDRGQIsGAJBmJaagZIy2d4Chfq22Hqy5ztso/edit?usp=sharing

Reply:
hey @finp I copied this from the push index.adoc, that link appears to go to a slide in a slide show which reads "flexible customer content"? Could you let me know a good example which I could copy the pattern from?

Reply:
also, I don't know if it's related but my index.adoc doesn't seem to be setting a title in the browser (neither does push which is what I copied from) can you let me know how I set a title?

Reply:
@philbrookes  I raised https://github.com/aerogear/mobile-docs/issues/73

will follow up with team on this naming, so ignore me ;)
... for now

Reply:
I have figured out how to set the title now.